dfx.json

```diff
@@ -1,21 +1,17 @@
 {
   "canisters": {
-    "smiley_dapp_assets": {
-      "dependencies": [
-        "smiley_dapp"
-      ],
-      "frontend": {
-        "entrypoint": "src/smiley_dapp_assets/src/index.html"
-      },
-      "source": [
-        "src/smiley_dapp_assets/assets",
-        "dist/smiley_dapp_assets/"
-      ],
-      "type": "assets"
-    },
-    "smiley_dapp": {
-      "main": "src/smiley_dapp/main.mo",
-      "type": "motoko"
+  "internet_identity": {
```

Comment:
dfx refused to call the canister without this, even if I specify the ID and the candid file manually.
See https://dfinity.slack.com/archives/CGA566TPV/p1680196767594109

Reply:
Unfortunately snsdemo overwrites dfx.json so calling the script from the demo fails. Should I change the source of overwriting the dfx.json, or add the section from the script after overwriting?